packages/block-editor/src/components/block-mover/index.native.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { first, last, partial, castArray } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { ToolbarButton } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { withInstanceId, compose } from '@wordpress/compose';
+
+const BlockMover = ( {
+	isFirst,
+	isLast,
+	onMoveDown,
+	onMoveUp,
+	firstIndex,
+	lastIndex,
+} ) => (
+	<>
+		<ToolbarButton
+			title={ ! isFirst ?
+				sprintf(
+					/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
+					__( 'Move block up from row %1$s to row %2$s' ),
+					firstIndex + 1,
+					firstIndex
+				) :
+				__( 'Move block up' )
+			}
+			isDisabled={ isFirst }
+			onClick={ onMoveUp }
+			icon="arrow-up-alt"
+			extraProps={ { hint: __( 'Double tap to move the block up' ) } }
+		/>
+
+		<ToolbarButton
+			title={ ! isLast ?
+				sprintf(
+					/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
+					__( 'Move block down from row %1$s to row %2$s' ),
+					lastIndex + 1,
+					lastIndex
```

Comment:
On the first block, the move down button is read by VoiceOver as: "Move block down from row 1 to row 0".
It should say "... from row 1 to row 2". 

The move up button is correct 👍

Reply:
Thanks, for noticing, should be fixed now :)